Artifacts/windows-visualstudio2015/artifact.ps1

```diff
@@ -0,0 +1,93 @@
+Param(
+    [Parameter(Mandatory=$true)]
+    [string] $sku,
+
+    [Parameter()]
+    [string] $installerArgs
+)
+
+function DownloadToFilePath ($downloadUrl, $targetFile)
+{
+    Write-Output ("Downloading installation files from URL: $downloadUrl to $targetFile")
+    $targetFolder = Split-Path $targetFile
+
+    if((Test-Path -path $targetFile))
+    {
+        Write-Output "Deleting old target file $targetFile"
+        Remove-Item $targetFile | Out-Null
```

Comment:
Consider using `-Force` so the call won't fail, especially since you are hiding the output.

Reply:
good call